syntax = "proto3";

message Request {
  string request_id = 1;
}

message Reply {
  string msg = 1;
}

service Pinger {
  rpc LevelOne (Request) returns (Reply) {}
  rpc LevelTwo (Request) returns (Reply) {}
  rpc Ping (Request) returns (Reply) {}
}
